MIGRATION-V1.md

```diff
@@ -0,0 +1,81 @@
+# Migration from 0.1.x to 1.0.x
+
+Upgrading existing `dynatrace-gcp-function` installations (either K8S or Cloud Function deployment) from 0.1.x is no supported.
+Existing old 0.1.x deployments need to be deleted and new 1.0.x installation needs to be deployed.
+New wersion (1.x.x) of `dynatrace-gcp-function` is using [Extensions 2.0](https://www.dynatrace.com/support/help/extend-dynatrace/extensions20/) and requires Dynatrace version 1.229 or higher.
+
+## K8S deployment
+Uninstall old helm release and install new wersion as described in [Dynatrace documentation](https://www.dynatrace.com/support/help/shortlink/deploy-k8#type)
+
+## Cloud Function deployment
+Run uninstall script:
+```shell script
+wget https://raw.githubusercontent.com/dynatrace-oss/dynatrace-gcp-function/master/scripts/uninstall.sh -O uninstall.sh ; chmod a+x uninstall.sh ; ./uninstall.sh
+```
+Remove old configuration file (actovation-config.yaml)
+Install new `dynatrace-gcp-function` deployment following instructions in [Dynatrace documentation](https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-cloud-platforms/google-cloud-platform/set-up-integration-gcp/deploy-as-gcp-function/) 
+
+## Dynatrace environment cleanup
+There might be some Dynatrace dashboards and/or alerts left from previous 0.1.x installations. They need to be deleted manually.
+New installation o 1.0.x version will add new dashboards and alerts.
+
+## List of incompatible changes
+Following GCP metrics dimension names has changed in 1.0.x. If you created own dashboards or alerts based on GCP metrics you need to update them.
```

Comment:
Please add visual indicators for If you created own dashboards or alerts based on GCP metrics you need to update them. 

⚠️ If you created own dashboards or alerts based on GCP metrics you need to update them